criu/include/magic.h

```diff
@@ -96,6 +96,7 @@
 #define FILES_MAGIC		0x56303138 /* Toropets */
 #define MEMFD_INODE_MAGIC	0x48453499 /* Dnipro */
 #define TIMENS_MAGIC		0x43114433 /* Beslan */
+#define PIDNS_MAGIC		0x12345678
```

Comment:
TODO: create correct magic

Reply:
You can use 0x61157326 /* Surgut */ or you can choose any other city that you like.
I think you can even choose a German city. For example, it can be 0x52311324 /* Berlin */, but in this case, you need to fix a comments which is right before the list of this constants.